tests/foreman/api/test_activationkey.py

```diff
@@ -188,8 +184,6 @@ def test_positive_update_name(self):
 
         :expectedresults: Activation key is created, and its name can be
             updated.
-
-        :CaseImportance: High
```

Comment:
Since the `CaseImportance` is defined in a global scope, you are removing it from here ??

Reply:
Its defined at the module scope, L13